docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -346,6 +346,10 @@ Total time spent throttling operations.
 (integer)
 Total time in milliseconds
 spent throttling operations.
+
+`write_load_avg`::
+(double)
+Average number of processors used while indexing documents.
```

Comment:
I think this is threads rather than processors, with the difference being IO.
```suggestion
Average number of write threads used while indexing documents.
```